packages/components/src/modal/ModalContent.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from 'react';
+import { View, Dimensions, Platform } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import type { ModalContentProps } from './ModalTypes';
+import TouchableWithoutFeedback from '../TouchableWithoutFeedback';
+
+class ModalContent extends React.Component<ModalContentProps> {
```

Comment:
The **ModalContent** can be implemented like the functional component instead class.

Reply:
Good catch. Updated